sdk/batch/Azure.ResourceManager.Batch/tests/TestCase/BatchAccountCertificateCollectionTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Batch.Models;
+using Azure.ResourceManager.Batch.Tests.Helpers;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Batch.Tests.TestCase
+{
+    public class BatchAccountCertificateCollectionTests : BatchManagementTestBase
+    {
+        public BatchAccountCertificateCollectionTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        private async Task<BatchAccountCertificateCollection> GetAccountCollectionAsync()
+        {
+            var container = (await CreateResourceGroupAsync()).GetBatchAccounts();
+            var input = ResourceDataHelper.GetBatchAccountData();
+            var lro = await container.CreateOrUpdateAsync(WaitUntil.Completed, Recording.GenerateAssetName("testaccount"), input);
+            var account = lro.Value;
+            return account.GetBatchAccountCertificates();
+        }
+
+        [TestCase]
+        public async Task CertificateCollectionApiTests()
+        {
+            //1.CreateOrUpdate
+            var container = await GetAccountCollectionAsync();
```

Comment:
we should no longer call things `container`. Please rename this variable to `collection`

Reply:
fixed